MovieLibrary/MovieList.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+class MovieStorage: ObservableObject {
+    @Published  var selectedGenre : Genre = .fantasy
+    @Published  var sortingOption : SortingOption = .title
+    
+   private let movies = [
```

Comment:
looks like the issue with offset, code style is broken - need add one space ))))